UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles .cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Options;
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Configuration;
+
+namespace UKHO.ExchangeSetService.FulfilmentService.Services
+{
+    public class FulfilmentAncillaryFiles : IFulfilmentAncillaryFiles
+    {
+        private readonly IOptions<FileShareServiceConfiguration> fileShareServiceConfig;
+
+        public FulfilmentAncillaryFiles(IOptions<FileShareServiceConfiguration> fileShareServiceConfig)                                       
+        {
+            this.fileShareServiceConfig = fileShareServiceConfig;
+        }
+
+        public async Task<bool> CreateSerialEncFile(string batchId, string exchangeSetPath, string correlationId)
+        {
+            if (!string.IsNullOrWhiteSpace(exchangeSetPath))
+            {
+                string serialFilePath = Path.Combine(exchangeSetPath, fileShareServiceConfig.Value.SerialFileName);
+                CheckCreateFolderPath(exchangeSetPath);
+                CultureInfo cultureInfo = CultureInfo.InvariantCulture;
+                int weekNumber = cultureInfo.Calendar.GetWeekOfYear(DateTime.UtcNow, CalendarWeekRule.FirstFourDayWeek, DayOfWeek.Monday);
+                var serailFileContent = String.Format("GBWK{0:D2}-{1}   {2:D4}{3:D2}{4:D2}UPDATE    {5:D2}.00{6}\x0b\x0d\x0a",
```

Comment:
Kindly correct this typo "serailFileContent" to "serialFileContent"

Reply:
Resolved